webapp/channels/src/packages/mattermost-redux/src/action_types/files.ts

```diff
@@ -13,4 +13,9 @@ export default keyMirror({
     RECEIVED_FILES_FOR_POST: null,
     RECEIVED_UPLOAD_FILES: null,
     RECEIVED_FILE_PUBLIC_LINK: null,
+    RECEIVED_FILE_PREVIEWS: null,
+
+    START_UPLOADING_FILE: null,
+    UPDATE_FILE_UPLOAD_PROGRESS: null,
+    REMOVE_FILE_PREVIEWS: null,
```

Comment:
More nitpicking, but would you mind renaming these to be written more like `STARTED_FILE_UPLOAD` or `CANCELED_FILE_UPLOAD` to be a bit more consistent with all of our `RECEIVED_*` actions?